quirks/change-password-URLs.json

```diff
@@ -1,13 +1,17 @@
 {
     "500px.com": "https://web.500px.com/settings/account/security",
     "acorns.com": "https://app.acorns.com/settings/change-password",
+    "allianz.com.br": "https://www.allianz.com.br/alteracao-de-password-ecliente",
     "amctheatres.com": "https://www.amctheatres.com/amcstubs/account",
+    "anatel.gov.br": "https://apps.anatel.gov.br/AnatelConsumidor/ConsumidorEditar.aspx",
     "atlassian.com": "https://id.atlassian.com/manage-profile/security",
     "autodesk.com": "https://accounts.autodesk.com/Profile/Security",
     "censys.io": "https://censys.io/account",
+    "claro.com.br": "https://minhanet.net.com.br/webcenter/portal/MinhaNet/pages_alterarsenha",
```

Comment:
This is a CP page on a different domain. Is this always right, or only for some accounts?

Reply:
Always right @rmondello Works for net.com.br, claro.com.br and netcombo.com.br because works like a SSO binding for auth.netcombo.com.br